solidus_i18n.gemspec

```diff
@@ -1,31 +1,29 @@
-# coding: utf-8
-lib = File.expand_path('../lib/', __FILE__)
-$LOAD_PATH.unshift lib unless $LOAD_PATH.include?(lib)
-
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'solidus_i18n/version'
 
-Gem::Specification.new do |s|
-  s.platform    = Gem::Platform::RUBY
-  s.name        = 'solidus_i18n'
-  s.version     = SolidusI18n.version
-  s.summary     = 'Provides locale information for use in Solidus.'
-  s.description = s.summary
+Gem::Specification.new do |spec|
+  spec.name        = 'solidus_i18n'
+  spec.version     = SolidusI18n.version
+  spec.authors      = ['Thomas von Deyen']
+  spec.email       = ['tvd@magiclabs.de']
+
+  spec.summary     = 'Provides locale information for use in Solidus.'
+  spec.description = 'A collection of translations for Solidus.'
+  spec.homepage    = 'https://solidus.io'
+  spec.license     = 'BSD-3-Clause'
 
-  s.author      = 'Thomas von Deyen'
-  s.email       = 'tvd@magiclabs.de'
-  s.homepage    = 'https://solidus.io'
-  s.license     = 'BSD-3'
+  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
 
-  s.files        = `git ls-files`.split("\n")
-  s.test_files   = `git ls-files -- spec/*`.split("\n")
```

Comment:
Shouldn't this line avoid the issue described?

Reply:
no, apparently it doesn't. Please also see https://github.com/rubygems/rubygems/issues/735

Reply:
You are right, thanks for pointing that out.